### PR TITLE
DOCSUP-8940: Edit and translate to Russian (deltaSumTimestamp)

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation (edit and translate to Russian)

### CODE REVIEW

File: docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+toc_priority: 141
+---
+
+# deltaSumTimestamp {#agg_functions-deltasum}
+
+Прибавляет разницу между последовательными строками. Если разница отрицательна — она будет проигнорирована. 
+
+Эта функция предназначена в первую очередь для [материализованных представлений](../../../sql-reference/statements/create/view.md#materialized), упорядоченных по некоторому временному бакету согласно timestamp, например, по бакету `toStartOfMinute`. Строки в таком материализованном представлении будут иметь одинаковый timestamp. Невозможно, чтобы они были объединены в "правом" порядке. Эта функция отслеживает `timestamp` значений, которые она видит. Поэтому можно правильно упорядочить состояния во время слияния.
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md:9]:
```suggestion
Эта функция предназначена в первую очередь для [материализованных представлений](../../../sql-reference/statements/create/view.md#materialized), упорядоченных по некоторому временному бакету согласно timestamp, например, по бакету `toStartOfMinute`. Поскольку строки в таком материализованном представлении будут иметь одинаковый timestamp, невозможно объединить их в "правом" порядке. Функция отслеживает `timestamp` наблюдаемых значений, поэтому возможно правильно упорядочить состояния во время слияния.
```

File: docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+**Синтаксис**
+
+``` sql
+deltaSumTimestamp(value, timestamp)
+```
+
+**Аргументы**
+
+-   `value` — входные значения, должны быть типа [Integer](../../data-types/int-uint.md), или [Float](../../data-types/float.md), или [Date](../../data-types/date.md), или [DateTime](../../data-types/datetime.md).
+-   `timestamp` — параметр для упорядочивания значений, должен быть типа [Integer](../../data-types/int-uint.md) или [Float](../../data-types/float.md) или [Date](../../data-types/date.md) или [DateTime](../../data-types/datetime.md).
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md:22]:
```suggestion
-   `timestamp` — параметр для упорядочивания значений, должен быть типа [Integer](../../data-types/int-uint.md), или [Float](../../data-types/float.md), или [Date](../../data-types/date.md) или [DateTime](../../data-types/datetime.md).
```

File: docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+**Возвращаемое значение**
+
+-   Накопленная разница между последовательными значениями, упорядоченными по параметру `timestamp`.
+
+Тип: [Integer](../../data-types/int-uint.md) или [Float](../../data-types/float.md) или [Date](../../data-types/date.md) или [DateTime](../../data-types/datetime.md).
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md:28]:
```suggestion
Тип: [Integer](../../data-types/int-uint.md), или [Float](../../data-types/float.md), или [Date](../../data-types/date.md), или [DateTime](../../data-types/datetime.md).
```

File: docs/en/sql-reference/aggregate-functions/reference/deltasum.md [hunk 4]
```diff
@@ -6,9 +6,10 @@ toc_priority: 141
 
 Sums the arithmetic difference between consecutive rows. If the difference is negative, it is ignored.
 
-Note that the underlying data must be sorted in order for this function to work properly.
-If you would like to use this function in a materialized view, you most likely want to use the
-[deltaSumTimestamp](deltasumtimestamp.md) method instead.
+Note that the underlying data must be sorted for this function to work properly.
+
+!!! info "Note"
+    If you would like to use this function in a [materialized view](../../../sql-reference/statements/create/view.md#materialized), you most likely want to use the [deltaSumTimestamp](../../../sql-reference/aggregate-functions/reference/deltasumtimestamp.md#agg_functions-deltasumtimestamp) method instead.
```
Comment (adevyatova) [inline docs/en/sql-reference/aggregate-functions/reference/deltasum.md:12]:
```suggestion
    The underlying data must be sorted for this function to work properly. If you would like to use this function in a [materialized view](../../../sql-reference/statements/create/view.md#materialized), you most likely want to use the [deltaSumTimestamp](../../../sql-reference/aggregate-functions/reference/deltasumtimestamp.md#agg_functions-deltasumtimestamp) method instead.
```

File: docs/en/sql-reference/aggregate-functions/reference/deltasum.md [hunk 5]
```diff
@@ -6,9 +6,10 @@ toc_priority: 141
 
 Sums the arithmetic difference between consecutive rows. If the difference is negative, it is ignored.
 
-Note that the underlying data must be sorted in order for this function to work properly.
-If you would like to use this function in a materialized view, you most likely want to use the
-[deltaSumTimestamp](deltasumtimestamp.md) method instead.
+Note that the underlying data must be sorted for this function to work properly.
```
Comment (adevyatova) [inline docs/en/sql-reference/aggregate-functions/reference/deltasum.md:9]:
```suggestion
```

File: docs/ru/sql-reference/aggregate-functions/reference/deltasum.md [hunk 6]
```diff
@@ -6,6 +6,11 @@ toc_priority: 141
 
 Суммирует арифметическую разницу между последовательными строками. Если разница отрицательна — она будет проигнорирована.
 
+Чтобы эта функция работала должным образом, исходные данные должны быть отсортированы.
+
+!!! info "Примечание"
+    В [материализованном представлении](../../../sql-reference/statements/create/view.md#materialized) вместо этой функции рекомендуется использовать [deltaSumTimestamp](../../../sql-reference/aggregate-functions/reference/deltasumtimestamp.md#agg_functions-deltasumtimestamp).
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasum.md:12]:
```suggestion
    Чтобы эта функция работала должным образом, исходные данные должны быть отсортированы. В [материализованном представлении](../../../sql-reference/statements/create/view.md#materialized) вместо нее рекомендуется использовать [deltaSumTimestamp](../../../sql-reference/aggregate-functions/reference/deltasumtimestamp.md#agg_functions-deltasumtimestamp).
```

File: docs/ru/sql-reference/aggregate-functions/reference/deltasum.md [hunk 7]
```diff
@@ -6,6 +6,11 @@ toc_priority: 141
 
 Суммирует арифметическую разницу между последовательными строками. Если разница отрицательна — она будет проигнорирована.
 
+Чтобы эта функция работала должным образом, исходные данные должны быть отсортированы.
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasum.md:9]:
```suggestion
```

File: docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+---
+toc_priority: 141
+---
+
+# deltaSumTimestamp {#agg_functions-deltasumtimestamp}
+
+Прибавляет разницу между последовательными строками. Если разница отрицательна — она будет проигнорирована.
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/deltasumtimestamp.md:7]:
```suggestion
Суммирует разницу между последовательными строками. Если разница отрицательна — она будет проигнорирована. 
```

**Comment by sevirov** [comment]
@Avogar Please, review this PR.

**Review by Avogar — APPROVED** [review]

**Pull request merged**